jujuclient/jujuclienttesting/stub.go

```diff
@@ -204,6 +214,12 @@ func (c *StubStore) CurrentController() (string, error) {
 	return c.CurrentControllerFunc()
 }
 
+// PreviousController implements ControllersGetter.PreviousController.
```

Comment:
```suggestion
// PreviousController implements ControllersGetter.PreviousController.
```

Not a fan of this comment, copy the comment, it provides better information.